client/blocks/plan-storage/test/bar.jsx

```diff
@@ -0,0 +1,121 @@
+/** @format */
+
+jest.mock( 'lib/abtest', () => ( {
+	abtest: () => '',
+} ) );
+
+const translate = x => x;
+jest.mock( 'i18n-calypso', () => ( {
+	localize: Comp => props => (
+		<Comp
+			{ ...props }
+			translate={ function( x ) {
+				return x;
+			} }
+		/>
+	),
+	numberFormat: x => x,
+} ) );
```

Comment:
Just a minor observation that there seems to be a convention where dependency imports are first, followed by blocks like these.

Reply:
Thanks! I'll stick to that